sos/plugins/foreman.py

```diff
@@ -139,6 +139,14 @@ def setup(self):
             'ping -c1 -W1 localhost'
         ])
 
+        # Dynflow Sidekiq
+        self.add_cmd_output('systemctl list-units dynflow*',
+                            suggest_filename='dynflow_units')
+        self.add_cmd_output('systemctl status '
+                            '"system-dynflow\\x2dsidekiq.slice"',
+                            suggest_filename='dynflow_sidekiq_status')
```

Comment:
The status request can be handled by the `Plugin.add_service_status()` method that hooks into the `InitSystem` abstraction. Though it does look like we'll need to plumb `**kwargs` through so `suggest_filename` can be passed properly.